theano/sandbox/cuda/basic_ops.py

```diff
@@ -3229,6 +3229,16 @@ def c_code_cache_version(self):
 gpu_join = GpuJoin()
 
 
+class GpuSplit(tensor.Split, GpuOp):
+    def make_node(self, x, axis, splits):
+        assert isinstance(x.type, CudaNdarrayType)
```

Comment:
I know many ops do this, but it isn't the best. Can you replace that line with:

x=as_cuda_ndarray_variable(x)

We are tring to move ops to that better way.